### DISCUSSION TITLE
kubernetes client version question..

### DISCUSSION THREAD

**Discussion by crotielwf** [apache/airflow#31580] [Q&A]
I use AKS v1.25.6, Airflow 2.6.1

I want to update kubernetes-client for AKS 1.25.6

but there are limits for apache-airflow-providers-cncf-kubernetes 

> kubernetes | >=21.7.0,<24

should it be kubernetes client version < 24?

I want to update kubernetes python client version 23.6.0 to 25.3.0 for my aks v1.25.6
is it okay?
is there any problem for kubernetes >= 24.0?

**Comment by potiuk** [comment]
Feel free to attempt to upgrade it in PR.
